_includes/metadata.html

```diff
@@ -0,0 +1,158 @@
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.twitter_username -%}
+        {%- capture link -%}https://twitter.com/{{ site.twitter_username }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.medium_username -%}
+        {%- capture link -%}https://medium.com/@{{ site.medium_username }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.quora_username -%}
+        {%- capture link -%}https://www.quora.com/profile/{{ site.quora_username }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.blogger_url -%}
+        {%- capture link -%}{{ site.blogger_url }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.work_url -%}
+        {%- capture link -%}{{ site.work_url }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.wikidata_id -%}
+        {%- capture link -%}https://www.wikidata.org/wiki/{{ site.wikidata_id }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.strava_userid -%}
+        {%- capture link -%}https://www.strava.com/athletes/{{ site.strava_userid }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.keybase_username -%}
+        {%- capture link -%}https://keybase.io/{{ site.keybase_username }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.gitlab_username -%}
+        {%- capture link -%}https://gitlab.com/{{ site.gitlab_username }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.dblp_url -%}
+        {%- capture link -%}{{ site.dblp_url }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if site.stackoverflow_id -%}
+        {%- capture link -%}https://stackoverflow.com/users/{{ site.stackoverflow_id }}{%- endcapture -%}
+        {%- assign sameaslinks = sameaslinks | push: link -%}
+    {%- endif -%}
+    {%- if sameaslinks != blank -%}
+        {%- assign sameaslinks = sameaslinks | split: "" -%}
+    {%- endif -%}
+
+    <script type="application/ld+json">
+      {
+        "author":
+        {
+          "@type": "Person",
+          "name": "{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}"
+        },
+        "url":"{{ page.url | prepend: site.baseurl | prepend: site.url }}",
+        "@type":"WebSite",
+        {% if page.description -%}
+        "description":"{{ page.description }}",
+        {% else if site.description -%}
+        "description":"{{ site.description }}",
+        {%- endif -%}
```

Comment:
minor: let's inline this code block:
```suggestion
        "description": "{%- if page.description -%}{{ page.description }}{%- else if site.description -%}{{ site.description }}{%- endif -%}",
```